coil-base/src/main/java/coil/size/ViewSizeResolver.kt

```diff
@@ -38,19 +39,11 @@ interface ViewSizeResolver<T : View> : SizeResolver {
             val viewTreeObserver = view.viewTreeObserver
 
             val preDrawListener = object : ViewTreeObserver.OnPreDrawListener {
-
-                private var isResumed = false
-
                 override fun onPreDraw(): Boolean {
-                    if (!isResumed) {
-                        val width = view.width - view.paddingLeft - view.paddingRight
-                        val height = view.height - view.paddingTop - view.paddingBottom
-                        if (width > 0 && height > 0) {
-                            isResumed = true
-                            viewTreeObserver.removePreDrawListenerSafe(this)
-                            continuation.resume(PixelSize(width, height))
-                        }
-                    }
+                    viewTreeObserver.removePreDrawListenerSafe(this)
+                    val width = max(1, view.width - view.paddingLeft - view.paddingRight)
+                    val height = max(1, view.height - view.paddingTop - view.paddingBottom)
+                    continuation.resume(PixelSize(width, height))
```

Comment:
`!isResumed` check no longer needed?

Reply:
I don't believe so, but going to double check on a few test projects before merging. I added it to ensure we don't resume the coroutine twice, however it should be fine as long as `viewTreeObserver.removePreDrawListenerSafe(this)` is before `continuation.resume(PixelSize(width, height))`.

Reply:
Actually, I didn't think we did initially, but I'm going to leave it in for safety. It's to prevent resuming the continuation twice, which would throw an exception.